packages/features/ee/billing/stripe-billling-service.ts

```diff
@@ -55,7 +55,10 @@ export class StripeBillingService implements BillingService {
       success_url: successUrl,
       cancel_url: cancelUrl,
       metadata: metadata,
-    });
+      invoice_creation: {
```

Comment:
Tested this locally and confirmed the invoice is created in the Stripe dashboard. Stripe doesn't send emails in test mode, but this should send the invoice to the provided email